### PR TITLE
Add streaming ConvEmformer model

### PR DESCRIPTION
**Pull request by yaozengwei**
This PR implements using the streaming ConvEmformer model (See https://github.com/k2-fsa/icefall/pull/440).

### CODE REVIEW

File: sherpa/bin/conv_emformer_transducer_stateless/decode.py [hunk 1]
```diff
@@ -0,0 +1,246 @@
+# Copyright      2022  Xiaomi Corp.        (authors: Fangjun Kuang)
```
Comment (csukuangfj) [inline sherpa/bin/conv_emformer_transducer_stateless/decode.py:1]:
```suggestion
# Copyright      2022  Xiaomi Corp.        (authors: Zengwei Yao)
```

File: sherpa/csrc/rnnt_conv_emformer_model.cc [hunk 2]
```diff
@@ -0,0 +1,154 @@
...
+    const torch::Tensor &features, const torch::Tensor &features_length,
+    const torch::Tensor &num_processed_frames, State states) {
+  // It contains [torch.Tensor, torch.Tensor, List[List[torch.Tensor]]
+  // which are [encoder_out, encoder_out_len, states]
+  //
+  // We skip the second entry `encoder_out_len` since we assume the
+  // feature input are of fixed chunk size and there are no paddings.
+  // We can figure out `encoder_out_len` from `encoder_out`.
+  auto states_tuple = torch::ivalue::Tuple::create(
+    std::get<0>(states), std::get<1>(states));
```
Comment (csukuangfj) [inline sherpa/csrc/rnnt_conv_emformer_model.cc:73]:
```suggestion
  auto states_tuple = torch::ivalue::Tuple::create(
    states.first, states.second);
```
is shorter.

**Review by pkufool — APPROVED** [review]
It would be better to add a github CI workflow for this model too.

File: sherpa/csrc/rnnt_conv_emformer_model.cc (hunk 2 above)
Comment (yaozengwei) [reply sherpa/csrc/rnnt_conv_emformer_model.cc:73]: Thanks.

**Comment by yaozengwei** [comment]
> It would be better to add a github CI workflow for this model too.

Thanks. Done.

**Pull request merged**